test/utils/DateFormatter.test.ts

```diff
@@ -0,0 +1,63 @@
+import { expect } from 'chai';
+
+import { dateToLocalTimeString } from 'app/utils/DateFormatter';
+
+describe('DateFormatter', () => {
+
+    it('should throw invalid date error', () =>{
+
+        const date: Date = new Date('nonsense date value');
+        expect(() => dateToLocalTimeString(date)).throws('DateFormatter - Invalid date');
+
+    });
+
+    it('should return YYYY-MM-DD format for a date', () =>{
+
+        const date: Date = new Date('2020-10-30T23:00:00.000');
+        expect(dateToLocalTimeString(date)).to.equal('2020-10-30');
+
+    });
+
+    it('should return string for local date rather than UTC', () =>{
```

Comment:
How node knows what the local time is? Aren't these tests going to fail when time zone changed into GMT later this year?